web/src/test/resources/com/graphhopper/http/resources/cargo_bike.yml

```diff
@@ -19,8 +16,13 @@ priority:
   surface:
     sand: 0.5
   # prefer better tracks
-  track_type: { other: 1.1, grade1: 1.1 }
+  track_type: { other: 1.0, grade1: 1.0, "*": 0.9 }
   # prefer official bike routes
   bike_network: { other: 0.5 }
   # avoid all situations where we have to get off the bike
-  get_off_bike: 0.5
\ No newline at end of file
+  get_off_bike:
+    "true": 0.5
+  max_height:
+    "<2.3": 0
+  max_width:
+    "<1.2": 0
```

Comment:
I like this. Still readable and generic enough for future extension. :+1:

Reply:
Agreed!